### DISCUSSION TITLE
c is not color?

### DISCUSSION THREAD

**Discussion by haucongle** [mantinedev/mantine#3219] [Q&A]
### What package has an issue

@mantine/core

### Describe the bug

In this page: [https://mantine.dev/styles/style-props/](https://mantine.dev/styles/style-props/)
All Mantine components support the following props:
![image](https://user-images.githubusercontent.com/23431433/208818221-c3cbdca7-01bf-4fa3-b362-661f0ec15562.png)

This component `<Badge color=""` is working but `<Badge c=""`

Can you explain?

### What version of @mantine/hooks page do you have in package.json?

5.9.0

### Do you know how to fix the issue

None

### Are you willing to participate in fixing this issue and create a pull request with the fix

None

**Comment by rtivital** [comment]
`c` prop sets CSS `color`. `color` prop changes the component appearance based on variant, you can see the difference when you set these props.
